docs/components/tasklist/userguide/using-tasklist.md

```diff
@@ -33,10 +33,15 @@ You can filter tasks to see the following:
 - Assigned to me
 - Unassigned
 - Completed
+- Custom (after selecting custom filters)
 
 To apply filter, click on the selection field in the left panel and choose preferred option.
 
-Click on the order icon ![order-icon](img/order-icon.png) to order the tasks. You can arrange them by the date of creation, the due date, or the follow-up date.
+Additionally, you can click the custom filters icon ![custom-filters-icon](img/custom-filters-icon.png) to define personalised filters. For example, you can filter by assignment state, status, and process.
```

Comment:
> filter by assignment state, status

@vsgoulart What's the difference between assignment state and status? We can also mention dates and variables.
E.g. "_filter by task statuses, assignees, processes, dates, and variables._".

Reply:
> > filter by assignment state, status
> 
> @vsgoulart What's the difference between assignment state and status? We can also mention dates and variables. E.g. "_filter by task statuses, assignees, processes, dates, and variables._".

@volodymyr-melnykc Assignment state = assgined/unassgined
status = open/completed

> We can also mention dates and variables.
E.g. "filter by task statuses, assignees, processes, dates, and variables.".

The idea was to not mention all filters since we might more in the future. What about `For example, you can filter by assignment state, status, and process and more.`

Reply:
@vsgoulart Got it.
Looks good! I think we can drop the "and" before "process" to not repeat it 2 times.

Reply:
yep, you're right

Reply:
I just noticed I had added that already and @christinaausley removed it :sweat_smile: 
Any ideas on how we can improve this @christinaausley ?

Reply:
I talked with @volodymyr-melnykc and he agreed that the current state of the branch is ok. So I think we're good to merge this.